search-service/src/main/kotlin/com/egm/stellio/search/service/TemporalEntityAttributeService.kt

```diff
@@ -275,6 +276,28 @@ class TemporalEntityAttributeService(
             .collectList()
     }
 
+    fun getTemporalsCount(ids: Set<URI>, types: Set<String>, attrs: Set<String>): Mono<Int> {
```

Comment:
better name it `getCountForEntities` to have a name inline with the `getForEntities` method